### ISSUE TITLE
Search results should have exact matches at the top of the list

### ISSUE THREAD

**Issue by bear454** [openSUSE/open-build-service#7563]
**Is your feature request related to a problem? Please describe.**
When searching for a project or package, en exact match doesn't show up at the top of the list, but somewhere among other results. For example, when searching on https://build.suse.de for "Devel:PubCloud", the Devel:PubCloud project is the 12th search result.

**Describe the solution you'd like**
Exact match results should be elevated to the top of the list.

**Describe alternatives you've considered**

**Additional context**

**Comment by dmarcoux** [comment]
I agree with you @bear454. Thank you for submitting an issue regarding this.
